ci/golang.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
@crackcomm  - so this is kinda a hack, but it works and produces smf.a built by go build system - once i have all the testing stuff working / passing i will then port the demo programs in your repo to this.

Reply:
oh cool, these tests pass too.